.github/workflows/haskell.yml

```diff
@@ -67,10 +67,23 @@ jobs:
           archive-uri: https://cache.haskellworks.io/${{ env.CABAL_CACHE_VERSION }}/${{ runner.os }}/${{ matrix.cabal }}/${{ matrix.ghc }}
           skip: "${{ secrets.BINARY_CACHE_URI != '' }}"
 
+      - name: Unix packages
+        env:
+          LOCALSTACK_AUTH_TOKEN: ${{ secrets.LOCALSTACK_AUTH_TOKEN }}
```

Comment:
Do we need the environment variables here too?

Reply:
Only the `LOCALSTACK_AUTH_TOKEN`